sfge/src/test/java/com/salesforce/rules/unusedmethod/ConstructorsTest.java

```diff
@@ -184,19 +194,11 @@ public void innerConstructorUsedByOuter_expectNoViolation(String variable, Strin
     /**
      * Tests for cases where an inner class's constructor is called by a sibling inner class.
      *
-     * @param variable - The way the variable is declared.
      * @param constructor - The way the constructor is invoked.
      */
-    @CsvSource({
-        "OuterClass.InnerClass, OuterClass.InnerClass",
-        "OuterClass.InnerClass, InnerClass",
-        "InnerClass, OuterClass.InnerClass",
-        "InnerClass, InnerClass"
-    })
-    @ParameterizedTest(name = "{displayName}: Declared as {0}, constructed as new {1}()")
-    @Disabled
-    public void innerConstructorUsedBySiblingInner_expectNoViolation(
-            String variable, String constructor) {
+    @ValueSource(strings = {"OuterClass.InnerClass", "InnerClass"})
```

Comment:
As above, variable declaration didn't terribly matter, so it was hardcoded.

Reply:
Same as above. If we have a change in mechanism, the additional cases will be helpful to look for regressions.